librz/analysis/arch/ppc/ppc_analysis.h

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2022 Rot127 <unisono@quyllur.org>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef PPC_ANALYSIS_H
+#define PPC_ANALYSIS_H
+
+#define INSOPS   insn->detail->ppc.op_count
+#define INSOP(n) insn->detail->ppc.operands[n]
+#define OP_CNT   insn->detail->ppc.op_count
+#define IMM(x)   (ut64)(insn->detail->ppc.operands[x].imm)
+
+#define PPC_IN_BE_MODE (mode & CS_MODE_BIG_ENDIAN)
+// Capstone does not extract the BO or BI fields of instructions. So we do it manually.
+#define PPC_READ_BO_FIELD (((rz_read_ble32(insn->bytes, PPC_IN_BE_MODE)) & 0x03e00000) >> 21)
```

Comment:
We should open a new issue for this in capstone.

Reply:
https://github.com/capstone-engine/capstone/issues/1909